avogadro/rendering/solidpipeline.cpp

```diff
@@ -157,12 +160,37 @@ void SolidPipeline::end()
                  d->depthTexture, m_width, m_height);
   d->firstStageShaders.setUniformValue("inAoEnabled", m_aoEnabled ? 1.0f : 0.0f);
   d->firstStageShaders.setUniformValue("inAoStrength", m_aoStrength);
+  d->firstStageShaders.setUniformValue("inFogStrength", (m_fogStrength));
   d->firstStageShaders.setUniformValue("inEdStrength", m_edStrength);
+  d->firstStageShaders.setUniformValue("inFogEnabled", m_fogEnabled ? 1.0f : 0.0f);
```

Comment:
Here, set inFogStrength like this:

`d->firstStageShaders.setUniformValue("inFogStrength", m_fogEnabled ? m_fogStrength : 0.0f);`